build/media-suite_helper.sh

```diff
@@ -64,12 +65,12 @@ set_title() {
 }
 
 do_exit_prompt() {
-    if [[ -n $build32 || -n $build64 ]]; then
+    if [[ -n "$build32" || -n "$build64" ]]; then
         create_diagnostic
         zip_logs
     fi
     do_prompt "$*"
-    [[ -n $build32 || -n $build64 ]] && exit 1
+    [[ -n "$build32" || -n "$build64" ]] && exit 1
```

Comment:
Should I just do `-n "${build32}${build64}"`?

Reply:
Sure. It's a safety for running the commands on mintty on its own.